pontoon/base/middleware.py

```diff
@@ -40,6 +40,8 @@ def process_request(self, request):
         except KeyError:
             ip = request.META['REMOTE_ADDR']
 
+        ip = ip.strip()
```

Comment:
Yeah, I saw IP addresses starting with a single space not being caught, e.g. " abc.def.ghi.jkl".